repository/src/main/java/org/apache/atlas/repository/store/aliasstore/ESAliasStore.java

```diff
@@ -141,6 +141,23 @@ public boolean updateAlias(AtlasEntity.AtlasEntityWithExtInfo accessControl, Atl
         return true;
     }
 
+    public void rebuildAlias(AtlasEntity.AtlasEntityWithExtInfo accessControl) throws AtlasBaseException {
+        String aliasName = getAliasName(accessControl.getEntity());
+
+        Map<String, Object> filter;
+
+        if (PERSONA_ENTITY_TYPE.equals(accessControl.getEntity().getTypeName())) {
+            filter = getFilterForPersona(accessControl);
+        } else {
+            filter = getFilterForPurpose(accessControl.getEntity());
```

Comment:
I think this else is redundant as we are not supporting repair of Purpose alias at all